src/main/webapp/app/modules/provider/record/single-record-view.tsx

```diff
@@ -0,0 +1,287 @@
+import './single-record-view.scss';
+
+import React from 'react';
+import { Collapse, Button, CardBody, Card, CardTitle } from 'reactstrap';
+import { Translate } from 'react-jhipster';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { connect } from 'react-redux';
+import { IRootState } from 'app/shared/reducers';
+import { getProviderEntity } from 'app/entities/organization/organization.reducer';
+// @ts-ignore
+import BuildingLogo from '../../../../static/images/building.svg';
+// @ts-ignore
+import PeopleLogo from '../../../../static/images/people.svg';
+// @ts-ignore
+import ServiceLogo from '../../../../static/images/service.svg';
+
+export interface ISingleRecordViewProps extends StateProps, DispatchProps, RouteComponentProps<{ orgId: string }> {
+  record: any;
+}
+
+export interface ISingleRecordViewState {
+  serviceWidths: any[];
+  locationWidths: any[];
+  isOrganizationOpen: boolean;
+  isServicesOpen: boolean;
+  isLocationsOpen: boolean;
+}
+
+class SingleRecordView extends React.Component<ISingleRecordViewProps, ISingleRecordViewState> {
+  state: ISingleRecordViewState = {
+    serviceWidths: [],
+    locationWidths: [],
+    isOrganizationOpen: false,
+    isServicesOpen: false,
+    isLocationsOpen: false
+  };
+
+  toggleOrganization = () => this.setState({ isOrganizationOpen: !this.state.isOrganizationOpen });
+  toggleServices = () => this.setState({ isServicesOpen: !this.state.isServicesOpen });
+  toggleLocations = () => this.setState({ isLocationsOpen: !this.state.isLocationsOpen });
+
+  componentDidMount() {
+    this.props.getProviderEntity(this.props.match.params.orgId);
+  }
+
+  render() {
+    const { isOrganizationOpen, isServicesOpen, isLocationsOpen } = this.state;
+    const { organization } = this.props;
+    const locationsCount = organization && organization.locations ? organization.locations.length : 0;
+    const servicesCount = organization && organization.services ? organization.services.length : 0;
+
+    return (
+      <div className="background single-record-view">
+        <Button tag={Link} to="/" color="" className="d-none d-sm-block position-fixed">
+          <FontAwesomeIcon icon="angle-left" />
+          &nbsp;
+          <Translate contentKey="record.singleRecordView.back" />
+        </Button>
+        <div className="col-md-8 offset-md-2 pt-4">
+          <Card className="record-card p-2">
+            <CardTitle>
+              <div className="d-flex w-100 justify-content-between">
+                <div>
+                  <h3>
+                    <b>{organization.name}</b>
+                  </h3>
+                  <section className="services pt-2">
+                    {servicesCount > 0
+                      ? organization.services.map(service => (
```

Comment:
What happens when there is too many services to fit in space? Jakub solved this by using auto sizer or something.

Reply:
This is a basic view and as I said I will handle this behavior in the next iteration of work.